assets/manifests.json

```diff
@@ -1,3 +1,3 @@
 {
-    "servicenow": "0.3.4"
+    "zendesk": "0.3.0"
```

Comment:
Does https://github.com/mattermost/mattermost-app-zendesk/pull/94 need to be merged before a new, compatible release can be cut?

Reply:
@hanzei Yes I'm just creating this as a placeholder